src/ViewRequest.js

```diff
@@ -269,7 +286,18 @@ class ViewRequest extends React.Component {
         }
       </PaneMenu>
     );
+  }
 
+  renderRequest(request) {
+    const {
+      stripes,
+    } = this.props;
+
+    const patronGroupName = this.getPatronGroupName(request);
+    const getPickupServicePointName = this.getPickupServicePointName(request);
+    const requestStatus = get(request, ['status'], '-');
+    // TODO: Internationalize this
+    const isRequestClosed = requestStatus.startsWith('Closed');
```

Comment:
There's no i18n work here; this is reading a value presented from the server.

Reply:
That's good to know!